data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+			choice
+				`	"Maybe it's better I do leave them alone."`
+					goto "better"
+				`	"There must be something I can do."`
+					goto "understand"
+
+			label "deny me"
+			action
+				set "gegno word: reject"
+			`	After a brief delay, Adrauni turns to the Gegno, mutters something in their language, then responds to you. "I see. And what do you propose is the reasoning for that?"`
+			choice
+				`	"The way they act around me."`
+				`	"From what you've told me before."`
+					goto "before"
+				`	"I don't know."`
+					goto "unsure"
+
+			`	"Indeed, the Gegno are wary of you, who lacks caution." A few seconds, and Adrauni gives a strange look towards the Gegno. You aren't sure what it's thinking. The Gegno asks something in their language, but receives no response. A few more moments go by before Adrauni speaks again. "Despite that, you have found yourself in precarious situations we have not. We as a species embraced contact with the Gegno, however, you remain the only human here. Perhaps therein lies a notable implication of distinction."`
+			choice
+				`	"Are you saying there's something I can do?"`
+					goto "understand"
+				`	"I'm not sure I understand."`
+					goto "interest"
+
+			label "before"
+			`	"I see." Adrauni blinks slowly while staring at you. It's almost as if it is surprised, or at the very least, not expecting that response. "Wise it is that you follow my fortunate instruction. If it weren't as such, I may not be the one speaking with you in this moment. Or, perhaps, it would be no one." It then looks forward again as you continue on your stroll, leaving you to think on those words. "Unfortunately, the Quarg are unable to further assist you in this delicate matter. If you recall, we have attempted prior consel with the Gegno, yet bound they are in their ways, even should it mean incredulously opposing us Quarg."`
+			choice
+				`	"Maybe it's better I do leave them alone."`
+					goto "better"
+				`	"There must be something I can do."`
+					goto "understand"
+
+			label "unsure"
+			action
+				set "gegno word: unsure"
+			`	Adrauni turns to you briefly, staring at you blankly, then turns to the Gegno and mutters something in their language before responding to you. "And why is it that you are unsure?"`
+			choice
+				`	"I wasn't sure you could convince them."`
+					goto "convince"
+				`	"I don't know anything about them."`
+
+			action
+				set "gegno word: friend"
+			`	After a pause, Adrauni turns toward you, stares at you briefly, then looks back forward as the three of you continue through the ring. "Quite an important conclusion, is that not? Many times I have said this, that the Gegno are not what I would consider humans to posses knowledge of. Should you learn more, I'm afraid even then the Gegno do not contrastingly desire, what was the human word - camaraderie?" The Gegno then asks something in their language, and Adrauni begins a somewhat lengthy conversation with them.`
```

Comment:
There's a lot of staring going on. Maybe some variety, like "glances at you" would be good?